ci/build/test-wheels.sh

```diff
@@ -139,7 +139,7 @@ elif [[ "$platform" == "macosx" ]]; then
     "$PIP_CMD" install -q "$PYTHON_WHEEL"
 
     # Install the dependencies to run the tests.
-    "$PIP_CMD" install -q aiohttp aiosignal frozenlist 'pytest==7.0.1' requests proxy.py
+    "$PIP_CMD" install -q aiohttp aiosignal numpy frozenlist 'pytest==7.0.1' requests proxy.py
```

Comment:
wheel_test is failing on mac due to this missing dependency, fix it while i'm here

Reply:
maybe pick this line in a separate PR?

Reply:
it takes 3.5 hour for the mac build job to finish and green and the change is also small so let's merge together